config/stack_custom.yaml

```diff
@@ -51,6 +51,7 @@ netcdf:
   build: YES
   shared: YES
   enable_pnetcdf: NO
+  build_netcdf_cxx: YES
```

Comment:
Instead of adding this to every yaml file, the default is YES to build netcdf_cxx
Just define `disable_netcdf_cxx: YES` and in the `build_netcdf.sh`check if that is Y.  If not, proceed as usual.